src/server/services/landingpage/qgslandingpagehandlers.h

```diff
@@ -54,6 +54,13 @@ class QgsLandingPageHandler: public QgsServerOgcApiHandler
     QgsServerOgcApi::Rel linkType() const override { return QgsServerOgcApi::Rel::self; }
     const QString templatePath( const QgsServerApiContext &context ) const override;
 
+    /**
+     *  Returns the path prefix, default is empty. Also makes sure that not-empty
+     *  prefix starts with "/" (ex: "/mylandingprefix"
+     */
```

Comment:
I know we don't have a lot of `\since` in server code but we probably should :).

Reply:
It's not part of the API.

Reply:
Maybe add it to the settings description if I didn't already.